school-manager-api/src/main/resources/api.yml

```diff
@@ -0,0 +1,228 @@
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Classroom'
+              examples:
+                classroomExample:
+                  value:
+                    classRange: 1
+                    studentsNumber: 20
+                    classIndex: "A"
+        '404':
+          description: Classroom not found
+          content:
+            text/plain:
+              examples:
+                notFoundExample:
+                  value: Classroom not found
+  /create:
+    post:
+      summary: Create Classroom
+      operationId: createClassroom
+      tags:
+        - Classroom
+      requestBody:
+        description: Classroom data
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Classroom'
+            examples:
+              classroomExample:
+                value:
+                  classRange: 1
+                  studentsNumber: 20
+                  classIndex: "A"
+      responses:
+        '200':
+          description: Classroom created successfully
+          content:
+            text/plain:
+              examples:
+                createdExample:
+                  value: Classroom was created /classroom/1?index=A
+          links:
+            GetClassroom:
+              operationId: getClassroom
+              parameters:
+                classroomId: '$response.body#/classRange'
+                description: >
+                  The `classRange` value returned in the response can be used as
+                  the `classRange` parameter in `GET /{classRange}`.
+        '409':
+          description: Classroom already exists
+          content:
+            text/plain:
+              examples:
+                conflictExample:
+                  value: Classroom already exists
+      callbacks:
+        MyCallback:
+          '{$request.query.callbackUrl}':
+            post:
+              requestBody:
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/Classroom"
+              responses:
+                '200':
+                  description: Callback successful
+  /update:
+    put:
+      summary: Update Classroom
+      tags:
+        - Classroom
+      requestBody:
+        description: Classroom data
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Classroom'
+            examples:
+              classroomExample:
+                value:
+                  classRange: 1
+                  studentsNumber: 20
+                  classIndex: "A"
+      responses:
+        '200':
+          description: Classroom updated successfully
+          content:
+            text/plain:
+              examples:
+                updatedExample:
+                  value: Classroom updated successfully
```

Comment:
Same problem here . Please add schema to response everywhere as in example. It can be simple string